rep-0003.rst

```diff
@@ -332,6 +409,17 @@ Our intent with Python is to track the minimum version provided in the
 supported Ubuntu platforms, as well as survey other commonly used OS
 platforms that support ROS to determine a reasonable minimum target.
 
+Melodic
+~~~~~~~
+Ubuntu will most likely stop supporting Python 2 in release 20.04.  To make
+sure ROS will be able to support that version of Ubuntu, ROS Python packages
+starting with Melodic Morenia are highly encouraged to support both Python 2.7
+and Python 3.5.  During the development of Melodic there will be work undertaken
+to support rosdep keys for both Python 2 and Python 3 so ROS package developers
+can more easily test with either version of Python.
```

Comment:
I absolutely agree with the paragraph. Before this can be merged we need to make sure that we are actually committing to do this.

Reply:
:+1: 

Not sure we should mention what part of the work will be done during Melodic release cycle, as the infrastructure work for python2 and 3 support will likely be more involved than just providing rosdep keys. 
Also I wouldnt restrict to Python 3.5 here as most targeted platform will have 3.6

Reply:
Sure, I'll change the wording to "3.5 or greater".